ui/src/app/auth/auth.guard.spec.ts

```diff
@@ -4,7 +4,7 @@ import { AuthService } from "./auth-service"
 import { ActivatedRouteSnapshot, Router, RouterStateSnapshot} from "@angular/router"
 import { HttpClientTestingModule } from "@angular/common/http/testing"
 import { AuthServiceStub, RouterStub } from "../../../test/resource/mock-stubs"
-import { ENABLE_ROLES, OSMT_ADMIN, OSMT_CURATOR } from "./auth-roles"
+import {ACTION_ROLES, ENABLE_ROLES} from "./auth-roles"
```

Comment:
Are we using the ENABLE_ROLES import here?

Reply:
Yes, it's used here. 
![image](https://user-images.githubusercontent.com/2341184/188945231-34d7ae58-786f-45a1-8429-cd677f75fb5c.png)